<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1321</width>
    <height>1026</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Деконволюция свёртки методом максимумом энтропии</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>221</width>
      <height>631</height>
     </rect>
    </property>
    <property name="title">
     <string>Параметры сигнала</string>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>181</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>Купол №1</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>A =</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_A1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>76</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>σ =</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_q1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>μ =</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_n1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>181</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>Купол №2</string>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>A =</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_A2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>76</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>σ =</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_q2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>μ =</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_n2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>30</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>181</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>Купол №3</string>
     </property>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>A =</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_A3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>76</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>σ =</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_q3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>μ =</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_n3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>40</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>500</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>N =</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_N">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>500</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>50</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>540</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>fd =</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_fd">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>540</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="shym">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>580</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>580</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>%,шум =</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>660</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Импульсная характеристика</string>
    </property>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>A = </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>σ =</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_An">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_qn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="start_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>910</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Выполнить</string>
    </property>
   </widget>
   <widget class="QWidget" name="graph_input" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>981</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="graph_impulse" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>300</y>
      <width>981</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="graph_output" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>570</y>
      <width>981</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>770</y>
      <width>221</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Метод Хука-Дживса</string>
    </property>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X0 = </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_x0">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>0.1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ΔX = </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_delta_x">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>0.01</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>860</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Энергия разности:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_x_energy_delta">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>860</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1321</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
